client/blocks/image-editor/index.jsx

```diff
@@ -95,6 +104,15 @@ const ImageEditor = React.createClass( {
 
 			this.setDefaultAspectRatio();
 		}
+
+		if ( newProps.originalAspectRatio && ! isEqual( newProps.originalAspectRatio, originalAspectRatio ) ) {
+			if ( newProps.originalAspectRatio.width > MinimumImageDimensions.WIDTH &&
+				newProps.originalAspectRatio.height > MinimumImageDimensions.HEIGHT ) {
+				this.setState( {
+					isGreaterThanMinimumDimensions: true
```

Comment:
For now let's use the information from IMAGE_EDITOR_IMAGE_HAS_LOADED. I just tested this with a emulated slow connection and it looks like we don't allow editing of images that are in the process of being uploaded, so theoretically the values should be correct. @retrofox can you think of any cases where this is not true?

If we'd like to in a follow up PR we can populate originalAspectRatio using the media item.